mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -412,11 +414,15 @@ CLASS aif net/minecraft/entity/Entity
 	METHOD j getCameraPosVec (F)Lcrv;
 		ARG 1 tickDelta
 	METHOD j allowsPermissionLevel (I)Z
+	METHOD j onBubbleColumnTop (Z)V
+		ARG 1 downwards
 	METHOD k isBeingRainedOn ()Z
 	METHOD k setHeadYaw (F)V
 		ARG 1 headYaw
 	METHOD k updatePassengerPosition (Laif;)V
 		ARG 1 passenger
+	METHOD k onBubbleColumn (Z)V
```

Comment:
Sounds a bit vague to me

Reply:
Changed to `onBubbleColumnCollision` (in the same vein as `onLavaCollision`).

Also `onBubbleColumnTop` -> `onBubbleColumnSurfaceCollision`